frame/nomination-pools/rpc/src/lib.rs

```diff
@@ -0,0 +1,66 @@
+// This file is part of Substrate.
```

Comment:
As we are moving away from custom RPC implementations that just map to runtime apis, could you also please remove all the `rpc` code. We just need the runtime api and then we can use the `state_Call` rpc.

Reply:
@Szegoo can you please do this? Then we can merge this :)

Reply:
@bkchr Did it now :), sorry for the late response.